.env

```diff
@@ -0,0 +1 @@
+REACT_APP_PROJECTS_URL = "http://localhost:7072/api/v1/projects"
```

Comment:
Let's store just the host+port:
API_HOST = "http://localhost:7072"

Reply:
solved!